src/main/java/edu/ucsb/cs156/happiercows/controllers/UserCommonsController.java

```diff
@@ -99,26 +98,29 @@ public ResponseEntity<String> putUserCommonsByIdBuy(
 
         String body = mapper.writeValueAsString(userCommons);
         return ResponseEntity.ok().body(body);
-    }
+      }
 
-  @ApiOperation(value = "Sell a cow, totalWealth updated")
+  @ApiOperation(value = "Sell cows, totalWealth updated")
   @PreAuthorize("hasRole('ROLE_USER')")
   @PutMapping("/sell")
   public ResponseEntity<String> putUserCommonsByIdSell(
-          @ApiParam("commonsId") @RequestParam Long commonsId) throws NoCowsException, JsonProcessingException {
+    @ApiParam("commonsId") @RequestParam Long commonsId,
+    @ApiParam("numCows") @RequestParam int numCows) // Want to set 1 as the default when nothing is passed in
```

Comment:
This comment... it's hard to know what to make of it.   

Does it mean you want to do this but you haven't yet?  If so, that probably doesn't belong in the code, but rather in an issue on your kanban board.

or, else you should go ahead and do it before we merge this code.

If the comment means something else, help me understand what you are trying to convey.

Reply:
Also, do you mean "zero"?  There can't be 'nothing' passed in, since an int always has a value.

You might want to check for negative numbers, though, because otherwise, you could end up using the "sell cows" with a negative number to do some kind of weird buy cows hack, and that might introduce weirdness into the game that we don't want.